gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/resources/webroot/assets/css/login_flow.css

```diff
@@ -0,0 +1,630 @@
+.container {
+    --topBorderColor: #6A4FF7;
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 64px 80px;
+    gap: 24px;
+    margin: auto;
+    width: 600px;
+    top: 100px;
+    background: #FFFFFF;
+    border: 1px solid rgba(0, 0, 0, 0.12);
+    border-radius: 4px;
+    border-top: 5px solid var(--topBorderColor);
+}
+
+.header {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 0;
+    gap: 32px;
+    height: 231px;
+    flex: none;
+    order: 0;
+    align-self: stretch;
+    flex-grow: 0;
+}
+
+.section, .form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 0;
+    gap: 24px;
+    flex: none;
+    align-self: stretch;
+    flex-grow: 0;
+}
+
+.card-section {
+    gap: 12px;
+}
+
+.short-section {
+    display: flex;
+    flex-direction: row;
+}
+
+.logo {
+     --width: 222px;
+    width: var(--width);
+    height: auto;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+}
+
+.title {
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 900;
+    font-size: 32px;
+    line-height: 16px;
+    text-align: center;
+    color: #25213A;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+}
+
+.header-description {
+    font-family: 'Roboto', sans-serif;
```

Comment:
font-family should only be declared once at body level : 

```
body {
  font-family: 'Roboto', sans-serif;
}
```

Reply:
done